run_page/gpxtrackposter/poster.py

```diff
@@ -179,6 +182,38 @@ def __draw_footer(self, d):
                 style=value_style,
             )
         )
+
+        d.add(
+            d.text(
+                self.trans("SPECIAL TRACKS"),
+                insert=(65, self.height - 20),
+                fill=text_color,
+                style=header_style,
+            )
+        )
+
+        d.add(d.rect((65, self.height - 17), (2.6, 2.6), fill=self.colors["special"]))
+
+        d.add(
+            d.text(
+                f"Over {str(special_distance1)} km",
```

Comment:
we need cut the length here 
user maybe add special distance to 10.1111111111111km that will over the content